File: code.js

function main() {
  var n = +readline();
  var s = readline();
  var x = new Array(n);
  var y = new Array(n);
  for (i = 0; i < n; i++) {
    xy = readline().split(" ");
    x[i] = +xy[0];
    y[i] = +xy[1];
  }
  
  // solve here
  
  print(n);
  print(s);
  print(x[0]);
  print(y[n - 1]);  
}

function solve(n) {
  return 0;
}

main();

File: code.rb
def main()
	n = gets.to_i
	s = gets
	x = Array.new(n)
	y = Array.new(n)
	for i in 0...n do
		x[i] , y[i] = gets.split.map(&:to_i)
	end

	puts n
	puts s
	puts x[0]
	puts y[n - 1]
end


def solve(n)
	return 0
end

main()
